### PR TITLE
adds purescript-phaser

### PR DESCRIPTION
**Pull request by lfarroco**
https://github.com/lfarroco/purescript-phaser

### CODE REVIEW

File: src/groups/lfarroco.dhall [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{ purescript-phaser =
```
Comment (thomashoneyman) [inline src/groups/lfarroco.dhall:1]: In the package sets we don't use the `purescript-` prefix. Would you mind updating the name of the library in your `.dhall` file to be this?

```suggestion
{ phaser =
```

**Review by thomashoneyman — COMMENTED** [review]
Once you update the package name I can go ahead and get this merged. Sorry for the hassle!

File: src/groups/lfarroco.dhall (hunk 1 above)
Comment (lfarroco) [reply src/groups/lfarroco.dhall:1]: thanks for the guidance, will do!

**Comment by lfarroco** [comment]
Should I rename it on https://github.com/purescript/package-sets/pull/991/files#diff-c508188958b68ce78976e33b66f47762f45f4a7bc134c5781cc38ee9ee81589aR3287 too? It is also necessary to create a new entry on `purescript/registry`?

**Comment by thomashoneyman** [comment]
Oh, sorry, yes. It should be renamed in both places. (Running `nix-shell --command make` will also do it for you.)

**Pull request merged**